public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -56,7 +57,7 @@ export function AppChrome({ children }: Props) {
   const contentClass = cx({
     [styles.content]: true,
     [styles.contentChromeless]: state.chromeless,
-    [styles.contentWithSidebar]: isExtensionSidebarOpen && !state.chromeless,
+    [styles.contentWithSidebar]: isExtensionSidebarOpen && !state.chromeless && headerLevels !== 2,
```

Comment:
I am not sure what does `headerLevels !== 2` do here and couple lines bellow. Also what do you think about extracting `headerLevels === 2` into variable that has meaningful name for its purpose and using it everywhere.